storage_service/locations/views.py

```diff
@@ -46,6 +46,11 @@ def package_list(request):
     packages = Package.objects.all()
     return render(request, 'locations/package_list.html', locals())
 
+def package_fixity(request, uuid):
+    package = Package.objects.get(uuid=uuid)
+    log_entries = FixityLog.objects.filter(package=package)
```

Comment:
This is easier with `FixityLog.objects.filter(package__uuid=uuid)` unless you need the package as well.  See https://docs.djangoproject.com/en/dev/topics/db/queries/#lookups-that-span-relationships for details.  Objects also store [reverse-foreign-key-sets](https://docs.djangoproject.com/en/dev/topics/db/queries/#following-relationships-backward).